canonical-web-and-design/docs.ubuntu.com
# Features microstack and openstack charmed

## Done

- Add features to docs.ubuntu.com

## QA

https://docs-ubuntu-com-283.demos.haus has 2 new items for Microstack and Charmed Openstack, check against [copy doc](https://docs.google.com/document/d/1mcfPYByoIT_iFOot6fxqq_bXX9p9vENWkLt4-ZI22Qg/edit#heading=h.joy95qx4bo0u)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/9759

## Screenshots

<img width="1440" alt="Screenshot 2021-05-18 at 12 57 26" src="https://user-images.githubusercontent.com/14939793/118646841-9ecd3680-b7d8-11eb-9725-f9bb1d95ea33.png">
